CHANGELOG.rst

```diff
@@ -35,12 +44,8 @@ This document describes changes between each past release.
 
 - Remove JSON Patch content-type from accepted types on the viewset, since it is handled
   in a separate view (#1031).
-- Permission schema children fields are now set during initialization instead of on
-  deserialization (#1046).
 - Upgraded to Kinto-Admin 1.8.1
 - Configure the Kinto Admin auth methods from the server configuration (#1042)
-- Request schemas (including validation and deserialization) are now isolated by method
-  and endpoint type (#1047).
```

Comment:
Yeah, these are unreleased. They probably got there by a problem in some auto merge.